### ISSUE TITLE
Check that project with the same name doesn't exist in the EditProjectViewModel before saving

### ISSUE THREAD

**Issue by simonrozsival** [toggl/mobileapp#998]
Ref #960 

Current implementation doesn't query the database:

```csharp
[DependsOn(nameof(Name))]
public bool SaveEnabled => 
        !string.IsNullOrWhiteSpace(Name) && Encoding.UTF8.GetByteCount(Name) <= MaxProjectNameLengthInBytes;
```

We rely on the fact that the view is opened only if the project did not exist in suggestions, but user can alter the text afterwards. This should be changed to prevent duplicates.

**Comment by amulware** [comment]
Remember to check this with the final trimmed string.

**Comment by amulware** [comment]
Possibly same as #936

**Comment by simonrozsival** [comment]
I would say the issue you mention only concerns UI, but it is definitelly connected and could be done in a single PR.

**Comment by kornelijepetak** [comment]
Closed by #1034
